packages/pn-personafisica-webapp/src/component/Deleghe/ConfirmationModal.tsx

```diff
@@ -39,7 +39,11 @@ export default function ConfirmationModal({
         <Box mx={3} sx={{ height: '100%' }}>
           <Grid container item mt={4}>
             <Grid item xs={10}>
-              <Typography variant="h5" sx={{ fontSize: '18px', fontWeight: '600' }}>
+              <Typography
+                id="title-dialog"
```

Comment:
@leleOFA è possibile avere un it più parlante? si tratta della dialog di conferma per le deleghe e non rappresenta un titolo generico

Reply:
fatto